index.rst

```diff
@@ -1,14 +1,26 @@
-ess-notebooks - Example and template notebooks for neutron data reduction 
+ess-notebooks - Example and template notebooks for neutron data reduction
 =========================================================================
 
-European Spallation Source (ESS) electronic notebooks for neutron data reduction. 
+European Spallation Source (ESS) electronic notebooks for neutron data reduction.
 
 Documentation
 =============
 
+.. toctree::
+   :caption: Imaging
+   :maxdepth: 2
+
+   imaging/bragg-edge-imaging-2D
```

Comment:
Should we move the demo into this tree - or maybe have another section for the demos?